google/utils.go

```diff
@@ -229,6 +229,23 @@ func linkDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return false
 }
 
+// expandLabels pulls the value of "labels" out of a schema.ResourceData as a map[string]string.
+func expandLabels(d *schema.ResourceData) map[string]string {
```

Comment:
Gotcha. I assumed that a compute.Labels or something along the line existed. Nevermind my comment then.